### PR TITLE
Register potatopioneers.is-a.dev

### PR DESCRIPTION
**Pull request by ponfertato**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [X] The file is in the `domains` folder and is in the JSON format.
- [X] You have completed your website. 
- [X] The website is reachable.  
- [ ] You're not using Vercel or Netlify.  
- [X] The CNAME record doesn't contain `https://` or `/`.  
- [X] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://potatopioneers.netlify.app/ or https://potatopioneers.pages.dev/

### CODE REVIEW

File: domains/potatopioneers.is-a.dev [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "description": "potatoPioneer's website",
+    "repo": "https://github.com/potatoPioneers/potatopioneers.site",
+    "owner": {
+        "username": "ponfertato",
+        "email": "ponfertato@ya.ru"
+        "github": "ponfertato"
+        "gitlab": "ponfertato"
```
Comment (WilliamDavidHarrison) [inline domains/potatopioneers.is-a.dev:8]: Remove this line

File: domains/potatopioneers.is-a.dev [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+        "telegram": "ponfertato"
+        "vk": "ponfertato"
```
Comment (WilliamDavidHarrison) [inline domains/potatopioneers.is-a.dev:10]: Remove this line

File: domains/potatopioneers.is-a.dev [hunk 3]
```diff
@@ -0,0 +1,18 @@
+{
+    "description": "potatoPioneer's website",
+    "repo": "https://github.com/potatoPioneers/potatopioneers.site",
+    "owner": {
+        "username": "ponfertato",
+        "email": "ponfertato@ya.ru"
+        "github": "ponfertato"
+        "gitlab": "ponfertato"
+        "telegram": "ponfertato"
```
Comment (WilliamDavidHarrison) [inline domains/potatopioneers.is-a.dev:9]: Remove this line

File: domains/potatopioneers.is-a.dev [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+        "vk": "ponfertato"
+        "discord": "ponfertato"
+    },
+    "record": {
+        "CNAME": "potatopioneers.github.io/potatopioneers.site",
```
Comment (WilliamDavidHarrison) [inline domains/potatopioneers.is-a.dev:14]: You can't use a CNAME with other records. Also, your CNAME is invalid.

**Review by wdhdev — CHANGES_REQUESTED** [review]
Please fix your JSON, complete the review comments and rename your file to `potatopioneers.json`

File: domains/potatopioneers.is-a.dev (hunk 1 above)
Comment (ponfertato) [reply domains/potatopioneers.is-a.dev]: Removed information other than email address

File: domains/potatopioneers.is-a.dev (hunk 2 above)
Comment (ponfertato) [reply domains/potatopioneers.is-a.dev]: Removed information other than email address

File: domains/potatopioneers.is-a.dev (hunk 3 above)
Comment (ponfertato) [reply domains/potatopioneers.is-a.dev]: Removed information other than email address

File: domains/potatopioneers.is-a.dev (hunk 4 above)
Comment (ponfertato) [reply domains/potatopioneers.is-a.dev]: Change the CNAME record to the address provided by Cloudflare

**Comment by ponfertato** [comment]
All problems corrected, pending approval

**Review by wdhdev — CHANGES_REQUESTED** [review]
You can't have a CNAME record with other records. You need to also rename your file to `potatopioneers.json`

File: domains/potatopioneers.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "potatoPioneer's Website",
+    "repo": "https://github.com/potatoPioneers/potatopioneers.site",
+    "owner": {
+        "username": "ponfertato",
+        "email": "ponfertato@ya.ru"
+        "github": "ponfertato"
+        "discord": "ponfertato"
```
Comment (WilliamDavidHarrison) [inline domains/potatopioneers.json:8]: Please remove this field.

File: domains/potatopioneers.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
+{
+    "description": "potatoPioneer's Website",
+    "repo": "https://github.com/potatoPioneers/potatopioneers.site",
+    "owner": {
+        "username": "ponfertato",
+        "email": "ponfertato@ya.ru"
+        "github": "ponfertato"
```
Comment (WilliamDavidHarrison) [inline domains/potatopioneers.json:7]: Please remove this field.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/potatopioneers.json (hunk 5 above)
Comment (ponfertato) [reply domains/potatopioneers.json]: Removed information on discord

File: domains/potatopioneers.json (hunk 6 above)
Comment (ponfertato) [reply domains/potatopioneers.json]: Removed information on github

**Review by wdhdev — APPROVED** [review]

**Pull request merged**